ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc

```diff
@@ -40,7 +43,13 @@ static bool NGraphPlacementRequested(const Node* node) { return true; }
 // Main entry point for the variable-capture.
 //
 Status CaptureVariables(Graph* graph, std::set<string> skip_these_nodes) {
-  const static std::map<
+  auto ReplaceVariableWithIdentityInfo =
+      std::bind(ReplaceVariable, _1, _2, _3, _4, _5, _6, _7, _8, _9, _10,
+                skip_these_nodes);
+  // This map is no longer static, since we need to do a partial application on
+  // ReplaceVariable. Note it is static in RewriteForTracking, since there the
+  // partial application is done with a constant argument that will not change
```

Comment:
Does this comment have to do with binding?

Reply:
bind creates a std::function that is resolvable only at runtime (since `skip_these_nodes` is unknown before hand), hence we cannot use it to initialize the map

parallel situation is:
```
x = rand()
static map<int, int> x = {1:2, 3:x};
```